clusterloader2/pkg/measurement/common/probes/manifests/dnsLookup/dns-prober-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: probes
+  name: dns
+  labels:
+    probe: dns
+spec:
+  selector:
+    matchLabels:
+      probe: dns
+  replicas: {{.Replicas}}
+  template:
+    metadata:
+      labels:
+        probe: dns
+    spec:
+      containers:
+        - name: dns
+          image: gcr.io/k8s-testimages/probes:v0.0.3
+          args:
+            - --metric-bind-address=0.0.0.0:8080
+            - --mode=dns
+            # Instead of creating dedicated "null-service" use one that's already exists
+            - --dns-probe-url=ping-server.probes
```

Comment:
I'm wondering whether we don't need fully qualified name..

Reply:
According to https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/ this should be fine.

Reply:
I agree it works - but I remember in the past there were significant performance differences when using fully qualified vs not fully qualified names.
Maybe add a TODO to evalue the impact of having that non-fully-qualified.

Reply:
A good idea may be to actually check both of them.

Reply:
Agree - can you please add a TODO?